gateway/server.go

```diff
@@ -141,11 +141,12 @@ func main() {
 
 	r.HandleFunc("/system/alert", faasHandlers.Alert)
 
-	r.HandleFunc("/system/function/{name:[-a-zA-Z_0-9]+}", queryFunction).Methods(http.MethodGet)
+	r.HandleFunc("/system/function/{name:[-a-zA-Z_0-9]+}", queryFunction).
```

Comment:
I'm not sure what this was added for. The system API was not supposed to be changed or covered by this work.

Similar to the other comments - none of the backends know what to do with this.

The PR I was expecting was to allow the functions to respond to the additional HTTP verbs.